### PR TITLE
move kinesis replays endpoint to explicit subpath

### PR DESCRIPTION
**Pull request by nemreid**
**Summary:** Summary of changes

## Changes

*   - Moved kinesis replay functionality from `/replays` to `/replays/kinesis`. Due to the use of explicit pathing to infer the type of the replay request, the `type` parameter is no longer required in the payload of a POST request.

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: packages/api-client/src/replays.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const postKinesisReplays = async (params: {
       headers: {
```
Comment (botanical) [inline packages/api-client/src/replays.ts:29]: Do we still expect the payload above to be MigrationCountsPayload? I think we can create a new type and add `kinesisStream`

**Comment by botanical** [comment]
Do we need to update `routes.js` as well?

**Comment by botanical** [comment]
Based off our discussion, I've stopped reviewing this PR and we can continue this work at some other point

**Comment by nemreid** [comment]
Closing in favor of rescope under CUMULUS-2632

**Pull request closed without merging**